momepy/functional/_diversity.py

```diff
@@ -176,6 +188,322 @@ def describe(
     return _compute_stats(grouper, q, include_mode)
 
 
+def values_range(
+    data: DataFrame | Series, graph: Graph, rng: tuple | list = (0, 100), **kwargs
+):
+    """Calculates the range of values within neighbours defined in ``graph``.
+    Uses ``scipy.stats.iqr`` under the hood.
```

Comment:
Remove the note if scipy iqr to avoid the confusion of keywords as they are still using rng.